src/main/java/com/alkemy/ong/controller/v1/UserController.java

```diff
@@ -13,4 +23,26 @@
 public class UserController {
 
     private final UserService service;
+
+    /**
+     * Actualiza firstName, lastName y photo de users.
+     *
+     * @param id Id del usuario que se quiere actualizar.
+     * @param patchDto Dto del usuario donde se encuentras los atributos que se pueden modificar.
+     * @return Dto actualizado.
+     * @throws Exception Devuelve un error 404.
+     */
+    @PatchMapping(V_1_PATCH_USERS)
+    public ResponseEntity<?> userPatch(
```

Comment:
It is not in the acceptance criteria to return a body as a response, could you please make the method void and add a 204 response status (NO_CONTENT). Like:

```java
@ResponseStatus(value= HttpStatus.NO_CONTENT)
public ResponseEntity<?> userPatch(...)
```